rules/tools/kiteshield/kiteshield.yara

```diff
@@ -36,7 +34,19 @@ rule kiteshield : high {
     $loader_s6 = {cf c0 da c2 c2 ca dc cd}
     // "0123456789abcdef"
     $loader_s7 = {b3 b5 b7 b5 b3 bd bf bd b3 b5 ec ec ec f4 f4 f4}
+    // Elf Magic
+    $elf_magic = {7f 45 4c 46}
+    // ET_EXEC
+    $et_exec = {02 00}
+    // EM_X86_64
+    $em_x86_64 = {3e 00}
+    // EM_AARCH64
+    $em_aarch64 = {b7 00}
 
   condition:
-    $loader_jmp and all of ($loader_s*) and elf.type==elf.ET_EXEC and elf.machine == elf.EM_X86_64
+    $loader_jmp 
+    and all of ($loader_s*) 
+    and $elf_magic at 0
```

Comment:
Tested this:
```
$ yara rules/tools/kiteshield/kiteshield.yara /bin/gofmt -c
warning: rule "kiteshield" in rules/tools/kiteshield/kiteshield.yara(40): string "$et_exec" may slow down scanning
warning: rule "kiteshield" in rules/tools/kiteshield/kiteshield.yara(42): string "$em_x86_64" may slow down scanning
warning: rule "kiteshield" in rules/tools/kiteshield/kiteshield.yara(44): string "$em_aarch64" may slow down scanning
1
```

Reply:
Better implementation: [`fc00476` (#254)](https://github.com/chainguard-dev/bincapz/pull/254/commits/fc0047621303b5150d65c72c322bece5b8ecd3ed)